tokens.xml

```diff
@@ -9837,6 +10019,62 @@ Whenever a card is put into your graveyard from anywhere, you may return it to y
             <token>1</token>
             <tablerow>1</tablerow>
         </card>
+        <card>
+            <name>Day</name>
+            <prop>
+                <type>State</type>
+                <maintype>State</maintype>
+            </prop>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/9/c/9c0f7843-4cbb-4d0f-8887-ec823a9238da.jpg?1630606483">MID</set>
+            <related attach="attach">Night<related>
```

Comment:
I don't think we want attach here, rather just have both make the other side, then destroy the original

Reply:
I was unsure on whether to have it attach or not. I'll let some others state what they think about it as well.

Reply:
Having a quick look here...
Wondering why you have "Day" `reverse-related` and ` related` to "Night", but "Night" has no related tags to "Day". 🤔

Reply:
If Day is reverse related and related to Night, Night does not need those tags. Both will be able to create the other. Adding the tags again to Night would be completely redundant.

Reply:
the question is whether or not we want to have the "night" token attach to "day" on creation, in a similar fashion to transformed cards (which would be inconsistent when the "night" side is created alone) or we could have the token simply create the other side and then have the user spend another action cleaning up "day" (I think some users might prefer this as it looks a bit cleaner)

also relevant: we talked about adding the option to hide the parent card when creating an attached card, this would appear as if the parent card has transformed into the attached token, this would require a minor addition to the protocol

Reply:
>If Day is reverse related and related to Night, Night does not need those tags. Both will be able to create the other. Adding the tags again to Night would be completely redundant.

Sure, let me rephrase the question:
Why don't you have "Night" reverse-related in the "Day" entry the same way as you have "Day" reverse-related in the "Night" entry already. That's how entries are normally structured in the file?
I just realized that you vary from the norm here and chose a different way of representing two very similar token.

Reply:
If we want them to be reverse related to each other instead of related and reverse related on day that's fine. It has zero impact on functionality.

Reply:
It should be uniform for Day and Night, one way or the other.

We have not a strict pattern setup yet for e.g. token that create other token (e.g. Emblems).
Normally, we reverse-connect token->card. For the token->token case it's mixed.

Reply:
I think we should work towards making it consistent then. There are currently about 9 or so token relations that use related instead of reverse related. A separate PR changing all of these to use reverse related could be good.

Reply:
>I think we should work towards making it consistent then. There are currently about 9 or so token relations that use related instead of reverse related. A separate PR changing all of these to use reverse related could be good.

I fully agree here. I try to open an issue to track that later.

The file is organically grown and since @Psithief was mostly editing it alone, there was never a logic description made or needed.

I did compile some basic things in a [first, simple documentation](https://github.com/Cockatrice/Magic-Token/wiki#documentation) to explain the features and possibilities. If there is more, please add it!
There is nothing about the structure and ordering yet. Or patterns like card->token, token->token.

Reply:
Thanks for opening https://github.com/Cockatrice/Magic-Token/pull/119, we can continue further discussion there.

Back to topic now: 😃 

>the question is whether or not we want to have the "night" token attach to "day" on creation, in a similar fashion to transformed cards (which would be inconsistent when the "night" side is created alone) or we could have the token simply create the other side and then have the user spend another action cleaning up "day" (I think some users might prefer this as it looks a bit cleaner)
>
>also relevant: we talked before about adding the option to hide the parent card when creating an attached card, this would appear as if the parent card has transformed into the attached token, this would require a minor addition to the protocol

I want to mention https://github.com/Cockatrice/Cockatrice/issues/4081 here.
There was the idea to have something like `<reverse-related attach="transform">` and allow for an even smoother support of double faced cards via card "turning over" like it would be done on a real table.
This would be a great application for Day/Night as well.